alertnow/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "cdb258cc-5e74-4fa2-be21-1489375bb370",
+  "app_id": "alertnow",
+  "classifier_tags": [
+    "Supported OS::Linux",
+    "Supported OS::Mac OS",
+    "Supported OS::Windows"
+  ],
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
```

Comment:
One more thing, can you add a `CHANGELOG.md` file?

Reply:
Yes I added this file.